driver/src/main/java/org/neo4j/driver/internal/packstream/PackStream.java

```diff
@@ -501,6 +501,29 @@ public double unpackDouble() throws IOException
             throw new Unexpected( "Expected a double, but got: " + toHexString( markerByte ));
         }
 
+        public byte[] unpackBytes() throws IOException
```

Comment:
Is this method different from `#unpackRawBytes()`?
Looks like only error message has changed. Can we remove `#unpackRawBytes()`?